common/data/src/main/java/com/quran/data/pageinfo/common/size/DefaultPageSizeCalculator.kt

```diff
@@ -8,12 +8,12 @@ open class DefaultPageSizeCalculator(displaySize: DisplaySize) : PageSizeCalcula
   private var overrideParam: String? = null
 
   override fun getWidthParameter(): String {
-    return when {
+    return overrideParam ?: when {
```

Comment:
this only matters for 1280 images (which is an override) where the 1024 would take precedence over the 1280, which is wrong in that case.